doc/manpage_ocamlformat.mld

```diff
@@ -256,6 +256,12 @@ OPTIONS (CODE FORMATTING STYLE)
            not fit on a single line. The default value is 2. Cannot be set in
            attributes.
 
+       --let-binding-punning={preserve|always|never}
+           Name  punning  in  extended  let bindings preserve uses let-punning
+           only when it exists in the source. always uses let-punning whenever
+           possible. never  never  uses  let-punning.  The  default  value  is
+           preserve.
```

Comment:
I know this is not the style of the current documentation, but I think an example of what let punning is should included.

Also regarding the name, I thought a let-binding is any use of `let`, but this only concerns bindings with let operators ? This does not have to be reflected in the name, but the documentation should probably say something.

Reply:
The problem is that it's not possible to add line breaks in the documentation, forcing the explanation to be concise. This is the output of cmdliner, which removes line breaks and reflow the text.

I'd be totally in favor of a new way of generating the list of option in the documentation and taking advantage of Odoc markup (paragraph, lists, text blocks). The cmdliner output must stay thought.